main.go

```diff
@@ -194,59 +205,37 @@ func ConnectTest() bool {
 // Get array of destinations zones
 func GetDSTZones(zoneID string) []string {
 	var DSTZones []string
-	site := fmt.Sprintf("https://%s/seg/api/v1/policies/visualization?matrixId=0&srcZoneId=%s", FSApplianceFQDN, zoneID)
-	method := "GET"
-
-	transport := &http.Transport{
-		TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
-	}
-
-	client := &http.Client{
-		Transport: transport,
-	}
-
-	req, err := http.NewRequest(method, site, nil)
-
-	if err != nil {
-		fmt.Println(err)
-		return nil
-	}
-	req.Header.Set("authority", FSApplianceFQDN)
-	req.Header.Set("user-agent", "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.114 Safari/537.36")
-	req.Header.Set("referer", fmt.Sprintf("https://%s/forescout-client/", FSApplianceFQDN))
-	user := fmt.Sprintf("%%22%s%%22", FSusername)
-	Cookies := fmt.Sprintf("JSESSIONID=%v; user=%v; XSRF-TOKEN=%v", JSESSIONID, user, XSRFTOKEN)
-	req.Header.Set("Cookie", Cookies)
-
-	res, err := client.Do(req)
-	if err != nil {
-		fmt.Println(err)
-		return nil
-	}
-	defer res.Body.Close()
-
-	body, err := ioutil.ReadAll(res.Body)
-	if err != nil {
-		fmt.Println(err)
-		return nil
-	}
-	fmt.Println(string(body))
+	body := buildRequest("/seg/api/v1/policies/visualization?matrixId=0&srcZoneId="+zoneID, "GET")
 	jsonParsed, err := gabs.ParseJSON(body)
 	if err != nil {
 		panic(err)
 	}
 	for _, child := range jsonParsed.Children() {
 		DSTZones = append(DSTZones, trimQuote(child.Path("dstZoneId").String()))
 	}
-
 	return DSTZones
 }
 
 // Get array of source zones
 func GetSRCZones(zoneID string) []string {
 	var SRCZones []string
-	site := fmt.Sprintf("https://%s/seg/api/v1/policies/visualization?matrixId=0&dstZoneId=%s", FSApplianceFQDN, zoneID)
-	method := "GET"
+	body := buildRequest("/seg/api/v1/policies/visualization?matrixId=0&dstZoneId="+zoneID, "GET")
```

Comment:
Please change calls like this to use `fmt.Sprintf` instead of `"endpoint"+var`. Like this:
```
body := buildRequest(fmt.Sprintf("/seg/api/v1/policies/visualization?matrixId=0&dstZoneId=%s",zoneID), http.MethodGet)
```